src/CVPresenter.ts

```diff
@@ -0,0 +1,97 @@
+import { IndexedFormula, NamedNode, Literal, Namespace } from "rdflib";
+import { ns } from "solid-ui";
+
+export interface Role {
+  startDate?: Literal,
+  endDate: Literal,
+  dates: string,
+  orgName: string,
+  roleText: string,
+  orgHomePage?: string
+}
+export interface CVPresentation { rolesByType: {
+    PastRole: Role[];
+    CurrentRole: Role[];
+    FutureRole: Role[];
+  };
+   skills: string[];
+  }
+
+const ORG = Namespace('http://www.w3.org/ns/org#');
+
+export const typesOfRole = ['PastRole', 'CurrentRole', 'FutureRole'];
```

Comment:
I think we should make the CVPresenter like CVCard with several mini functions outside of the component.

Reply:
I think you fixed that now